### PR TITLE
Refactor repository for readability and re-instantiation of all contracts 

### PR DESCRIPTION
**Pull request by NatalieChin80**
- Fixed javascript indentation 
 - Moved declaration of addresses to global scope in the contract 
 - Refactored `allAccounts[index]` to address name 
 - Changed parameters of `updateClosedChannel` and `close` tests 
 - Changed all tests to re-instantiate a new instantiation of all required Solidity Contracts 
 - assertRevert finally works now

### CODE REVIEW

File: test/AdditionalTokens.js [hunk 1]
```diff
@@ -1,212 +1,193 @@
...
-
-            },
-            "WETHToken": {
-                args:[initialCreation, "WETH", 18, "STK"],
-                "instanceOf": "ERC20Token",
-                "from": allAccounts[3]
+let userAddress = allAccounts[0];
+let recipientAddress = allAccounts[1];
+let signerAddress = allAccounts[2]; 
+let nonParticipantAddress = allAccounts[3];
```
Comment (thomasmetta) [inline test/AdditionalTokens.js]: Use const if the variable remain constant.

File: test/ERC20Token.js [hunk 2]
```diff
@@ -1,103 +1,89 @@
...
 const initialCreation = testConstant.INIT;
 const timeout = testConstant.TIMEOUT;
+let userAddress = allAccounts[0];
+let recipientAddress = allAccounts[1];
+let signerAddress = allAccounts[2];
+let nonParticipantAddress = allAccounts[3];
+const signersPk = Buffer.from(testConstant.SIGNER_PK, 'hex');
+const userPk = Buffer.from(testConstant.USER_PK, 'hex');
+const recipientPk = Buffer.from(testConstant.RECIPIENT_PK, 'hex');
+var nonce = 1;
```
Comment (thomasmetta) [inline test/ERC20Token.js]: Try to use let instead of var if the variable is going to change.

File: test/IllegalStateTransition.js [hunk 3]
```diff
@@ -1,446 +1,710 @@
...
         });
 
-        it("User cannot close channel with signer address signed transaction", async() =>
-        {
-            nonce++; 
-            const amount = 2;
+        var amount = 2;
 
-            const cryptoParams = closingHelper.getClosingParameters(ERC20Token.options.address,nonce,amount,MultiChannel.address,signersPk);
+        const properClose = closingHelper.getClosingParameters(ERC20Token.options.address, nonce, amount, MultiChannel.address, signersPk);
```
Comment (thomasmetta) [inline test/IllegalStateTransition.js]: Instead of ProperClose, can we name it relating signature? like ValidSignature

Comment (NatalieChin80) [reply test/IllegalStateTransition.js]: Fixed in 36bf110

**Pull request merged**